### PR TITLE
Update main get started page links

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #16046.

### CODE REVIEW

File: docs/core/tutorials/consuming-library-with-visual-studio.md [hunk 1]
```diff
@@ -89,11 +88,13 @@ Just as you included unit tests in the same solution as your class library, you
...
 
-## Distributing the library in a NuGet package
+## Distribute the library in a NuGet package
 
 You can make your class library widely available by publishing it as a NuGet package. Visual Studio does not support the creation of NuGet packages. To create one, you use the [`dotnet` command line utility](../tools/dotnet.md):
 
-1. Open a console window. For example in the **Ask me anything** text box in the Windows taskbar, enter `Command Prompt` (or `cmd` for short), and open a console window by either selecting the **Command Prompt** desktop app or pressing Enter if it's selected in the search results.
+1. Open a console window.
+
+  For example, enter **Command Prompt** in the search box on the Windows task bar. Select the **Command Prompt** desktop app or press **Enter** if it's already selected in the search results.
```
Comment (Youssef1313) [inline docs/core/tutorials/consuming-library-with-visual-studio.md]: The next list item shows as 1 as well. Markdown require more spaces (not sure if it's 3 or 4). I'd make them 4 spaces.

```suggestion
    For example, enter **Command Prompt** in the search box on the Windows task bar. Select the **Command Prompt** desktop app or press **Enter** if it's already selected in the search results.
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

File: docs/core/tutorials/consuming-library-with-visual-studio.md (hunk 1 above)
Comment (gewarren) [reply docs/core/tutorials/consuming-library-with-visual-studio.md]: Thanks for noticing this. It's fixed now.

**Comment by gewarren** [comment]
Let me know if you like having the videos separate or not.

https://review.docs.microsoft.com/en-us/dotnet/core/get-started?branch=pr-en-us-16089&tabs=macos#tutorials

**Comment by gewarren** [comment]
@BillWagner Could you review please?

**Comment by gewarren** [comment]
@BillWagner Can you take a look now that you're back (and when you have a minute)?

File: docs/core/tutorials/consuming-library-with-visual-studio.md [hunk 2]
```diff
@@ -6,24 +6,23 @@ ms.author: wiwagn
 ms.date: 06/05/2018
 ms.custom: "vs-dotnet, seodec18"
 ---
-
 # Consume a .NET Standard library in Visual Studio 2017
 
-Once you've created a .NET Standard class library by following the steps in [Building a C# class library with .NET Core in Visual Studio 2017](./library-with-visual-studio.md) or [Building a Visual Basic class library with .NET Core in Visual Studio 2017](vb-library-with-visual-studio.md), tested it in [Testing a class library with .NET Core in Visual Studio 2017](testing-library-with-visual-studio.md), and built a Release version of the library, the next step is to make it available to callers. You can do this in two ways:
+Once you've created a .NET Standard class library by following the steps in [Building a C# class library with .NET Standard in Visual Studio](./library-with-visual-studio.md) or [Building a Visual Basic class library with .NET Core in Visual Studio 2017](vb-library-with-visual-studio.md), tested it in [Testing a class library with .NET Core in Visual Studio 2017](testing-library-with-visual-studio.md), and built a Release version of the library, the next step is to make it available to callers. You can do this in two ways:
```
Comment (BillWagner) [inline docs/core/tutorials/consuming-library-with-visual-studio.md]: It looks like a couple "2017" keys stayed here. Do you want to fix them now, or wait until we update that content?

```suggestion
Once you've created a .NET Standard class library by following the steps in [Building a C# class library with .NET Standard in Visual Studio](./library-with-visual-studio.md) or [Building a Visual Basic class library with .NET Core in Visual Studio](vb-library-with-visual-studio.md), tested it in [Testing a class library with .NET Core in Visual Studio](testing-library-with-visual-studio.md), and built a Release version of the library, the next step is to make it available to callers. You can do this in two ways:
```

**Review by BillWagner — APPROVED** [review]
This LGTM @gewarren 

You can :shipit: when ready.

**Pull request merged**